templates/example-environments/rdo-cloud-env.yaml

```diff
@@ -8,3 +8,29 @@ parameters:
   undercloud_image: CentOS-7-x86_64-GenericCloud-1703
   undercloud_flavor: m1.large
   toci_jobtype: multinode-1ctlr-featureset004
+  undercloud_user_data: |
+    #!/bin/bash
+    rpm -q git || yum -y install git
+    git clone https://git.openstack.org/openstack-infra/tripleo-ci
+    export HOME=/root
+    tripleo-ci/scripts/tripleo.sh --repo-setup
+    yum -y install python-heat-agent*
+    rm -f /etc/yum.repos.d/delorean*
+    rm -rf tripleo-ci
+    systemctl daemon-reload
+    systemctl enable os-collect-config
+    reboot
```

Comment:
i think it may make sense to default this in the top level template instead of having to carry it in the environment.